tianshou/trainer/onpolicy.py

```diff
@@ -138,12 +147,11 @@ def onpolicy_trainer(
                     if writer and gradient_step % log_interval == 0:
                         writer.add_scalar(
                             k, stat[k].get(), global_step=gradient_step)
-                t.update(step)
                 t.set_postfix(**data)
             if t.n <= t.total:
                 t.update()
         # test
-        result = test_episode(policy, test_collector, test_fn, epoch,
+        test_result = test_episode(policy, test_collector, test_fn, epoch,
```

Comment:
is it a bug?

Reply:
```suggestion
        result = test_episode(policy, test_collector, test_fn, epoch,
```

Reply:
No, just for clear

Reply:
My bad, it indeed is a bug, will fix that